GOVERNANCE.md

```diff
@@ -45,18 +30,15 @@ be accepted unless:
   the change. Previously-objecting Collaborators do not necessarily have to
   sign-off on the change, but they should not be opposed to it.
 * The change is escalated to the CTC and the CTC votes to approve the change.
-  This should be used only after other options (especially discussion among
-  the disagreeing Collaborators) have been exhausted.
+  This should be used only after discussion among the disagreeing Collaborators
+  has been exhausted.
```

Comment:
Thanks, @gibfahn. Changed `be used` to `happen` but otherwise went with your suggestion:

> This should only happen if disagreements between Collaborators cannot be resolved through discussion.